### PR TITLE
create docs section for deployment info

### PR DESCRIPTION
**Pull request by stevecrozz**
The README file needs some more organization. So I'm starting by factoring out the deployment notes into individual files in the ./docs folder

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage. <a href='http://create-docs-section-for-deployment.23f39035947abfbf29f9f7741cb9c271.ttrcloud.com' target='_blank'>Click to open.</a>

File: docs/deployment/heroku.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+We designed Errbit to work well with Heroku. These instructions should result
+in a working deploy, but you should modify them to suit your needs:
+
+## Clone and prepare the source code repository
+```bash
+git clone git@github.com:errbit/errbit.git
+cd errbit
+```
+
+- Update `db/seeds.rb` with admin credentials for your initial login
```
Comment (shingara) [inline docs/deployment/heroku.md]: you need commit this change to be used on heroku

**Comment by shingara** [comment]
:+1:

File: docs/deployment/heroku.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+Commit the results:
+```bash
+git commit -m "Update db/seeds.rb with initial login"
+```
+
+## Create an app on Heroku and push the source code
+```bash
+gem install heroku
```
Comment (brianvanburken) [inline docs/deployment/heroku.md]: Heroku doesn't use the gem version anymore. Installing the gem will result in a deprecation message: 
```
 !    The `heroku` gem has been deprecated and replaced with the Heroku Toolbelt.
 !    Download and install from: https://toolbelt.heroku.com
 !    For API access, see: https://github.com/heroku/heroku.rb
```

Instead Heroku uses a command-line interface called 'heroku toolbelt' as described here: https://devcenter.heroku.com/articles/heroku-command

**Comment by arthurnn** [comment]
thats great

**Pull request merged**